packages/plugin-core/src/constants.ts

```diff
@@ -513,6 +513,11 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     title: `${CMD_PREFIX} Copy Codespace URL`,
     when: `${DendronContext.PLUGIN_ACTIVE} && shellExecutionSupported`,
   },
+  COPY_AS: {
```

Comment:
can we use `cmd|ctrl+k cmd|ctrl+c` as keyboard shortcut for this?

Reply:
By default, in windows, this is a shortcut to add a comment line
`{ "key": "ctrl+k ctrl+c",         "command": "editor.action.addCommentLine",`

Reply:
yep. i figured we're not going to win the battle of not conflicting with keyboard shortcuts. rather, just go for conflicts of items that are not used a lot. the `ctrl+/` shortcut is the main one used for comments and we found that people generally don't set shortcuts themselves. for new commands going forward, lets try to find a shortcut for them in the `ctrl+k/cmd+k` namespace that doesn't conflict with a popular command